libs/fades.liq

```diff
@@ -290,8 +292,8 @@ def crossfade(~duration=5.,~override_duration="liq_cross_duration",
               ~default=(fun (a,b) -> sequence([a, b])),
               ~high=-15., ~medium=-32., ~margin=4.,
               ~minimum=(-1.),~width=2.,~conservative=true,s)
-  fade.out = fade.out(type="sin",duration=fade_out)
-  fade.in  = fade.in(type="sin",duration=fade_in)
+  let fade.out = fade.out(type="sin",duration=fade_out)
+  let fade.in  = fade.in(type="sin",duration=fade_in)
```

Comment:
What is the scope is this `let`? Can it leak into the main record type?

Reply:
Usual scoping.

The instruction
```
let r.a = f
```
is a shorthand for
```
r = {{ r | a = f }}
```
so that the scoping conventions are as usual.

Reply:
Cool!